### PR TITLE
Add invoice items operation

### PR DESCRIPTION
**Pull request by mzishtiaq**
### Checklist

- [ ] Documentation follows the [Style Guide](https://mews.atlassian.net/wiki/x/KJAoCw)
- [ ] JSON examples updated
- [ ] Properties in JSON examples are in the same order as in property tables
- [ ] Changelog dated the day when PR merged
- [ ] Changelog accurately describes all changes
- [ ] Changelog highlights the affected endpoints or operations
- [ ] Changelog highlights any deprecations
- [ ] All hyperlinks tested
- [ ] Deprecation Table updated if any deprecations
- [ ] SUMMARY.md updated if new pages added

### CODE REVIEW

**Review by MirkoC — APPROVED** [review]

File: operations/invoiceitems.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# InvoiceItems
+
+## Get invoiceItem
```
Comment (MikeAdamsMews) [inline operations/invoiceitems.md:4]: I think the name should be a natural language name
```suggestion
## Get invoice item
```

File: operations/invoiceitems.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+A invoiceItem represents an item within invoice.
```
Comment (MikeAdamsMews) [inline operations/invoiceitems.md:6]:
```suggestion
This operation returns a single specified invoice item, which is an individual order item that is part of an invoice.
```

File: operations/invoiceitems.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+Note: This operation needs [Authentication](../essential-guide/authentication.md) and supports the following [JSON:API features](../essential-guide/features.md):
```
Comment (MikeAdamsMews) [inline operations/invoiceitems.md:7]:
```suggestion
The operation needs [Authentication](../essential-guide/authentication.md) and supports the following [JSON:API features](../essential-guide/features.md):
```

File: operations/invoices.md [hunk 4]
```diff
@@ -142,6 +143,14 @@ Below is a list of all possible fields this endpoint can return including relati
 | `createdAt` | string | required, max length 25 characters | Created at timestamp in RFC 3339 format. |
 | `updatedAt` | string | required, max length 25 characters | Updated at timestamp in RFC 3339 format. |
 
+#### invoice_item_relationships
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `product` | object | required | Details of the product associated with the invoiceItem. |
```
[the remark this answers is not in the document]
Comment (mzishtiaq) [reply operations/invoices.md:150]: All the current types are JSON schema primitive data types but we can discuss this further within the team.

**Pull request merged**